scripts/utils.js

```diff
@@ -294,25 +294,26 @@ function attachTooltip (anchor, tooltip, pos = 'right', time = 200) {
     })
 }
 
-function resetExtensionStorage () {
+function initDefaultOptions () {
   chrome.storage.sync.set({
```

Comment:
When you remove the extension from firefox and again install it, the welcome page will not be shown.